### PR TITLE
feat(Licensing): how to get a kommander license

### PR DESCRIPTION
**Pull request by brandonc**
Closes D2IQ-64510

## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-64510

## Description of changes being made

Adds details for accessing Kommander license from support downloads

## Checklist
- [x] Make sure you change all affected versions (e.g. 1.10, 1.11, 1.12, 1.13).

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing) for more information.

### CODE REVIEW

**Comment by cmastr** [comment]
@brandonc I had to merge some of your and @nLight's changes. Can both of you take a look at the proposed new Licensing stuff?

File: pages/ksphere/kommander/latest/licensing/index.md [hunk 1]
```diff
@@ -1,35 +1,28 @@
...
+title: License 
 excerpt:
 menuWeight: 6
 ---
 
-## Licensing
+## Purchase a License
 
-Kommander requires a valid license to continue use beyond adding a second cluster.
+Kommander requires a valid license if you want to add more than two clusters.
```
Comment (nLight) [inline pages/ksphere/kommander/latest/licensing/index.md]: Should we call out that Kommander cluster itself counts as a cluster?

> You can evaluate Kommander running two clusters for free. Kommander cluster plus one worker cluster. If you want to add more clusters you need a valid license.

Comment (samvantran) [reply pages/ksphere/kommander/latest/licensing/index.md]: +1

Comment (brandonc) [reply pages/ksphere/kommander/latest/licensing/index.md]: "worker cluster" is not a term I've heard before and I don't want to overload worker node pools. Went with "additional cluster"

**Pull request merged**